### DISCUSSION TITLE
"-fastflix-xxxx." output filename extension

### DISCUSSION THREAD

**Discussion by Battlestar1965** [cdgriffith/FastFlix#613] [Q&A] [answered]
Hello,

i just wanted to know how to get rid of the extension, that is always added to the end of the output filename like "-fastflix-25a9.mkv".
I couldn't find any switch for that in the setting...or i overlooked it. Help would be much appreciated.

**Comment by cdgriffith** [comment, marked as answer]
Currently this is a "hidden" option changed in the config file itself. https://github.com/cdgriffith/FastFlix/issues/553#issuecomment-2101163141

**Comment by Battlestar1965** [comment]
Thanks for your answer. Fun fact...  i already knew about it once, i just forgot about this "hidden" option. I'm getting old....
